### DISCUSSION TITLE
Map revamp

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1464] [Ideas]
Map related tickets:
* [ ] #1405  
* [ ] #1445 
* [ ] #1446 
* [ ] #1447  
* [ ] #1456

**Comment by WohnungNDS** [comment]
displaying disaggredated Data on the map works well... As you can see [here](https://wohnungnds.github.io/IM_Site/3-1-3/)
Using `remote_theme: brockfanning/open-sdg@map-disaggregation-select` #374 


![image](https://user-images.githubusercontent.com/67795376/139076294-f0710993-09ba-4e3e-9654-c65bdeba4834.png)

only two things:

1) the min may value is not set correctly. in the data I have numer and percent. the min max value of percent will be displayed even if i change the dropdown to number (AnzahI)

2) It would be nice to exclude some disaggregations, because in some indicators I have a lot of disaggregations that today don't make sense to have it on the map but may be in future... so it youd be nice to exclude some of them

![image](https://user-images.githubusercontent.com/67795376/139079320-6af74f71-d561-413b-b256-b9ad31fb677c.png)

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin perhaps you can add #374 and #356 in the lust above....

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin 
Another question may be to preset the displayed year in the timeslider. I have more or loss long time series (15 years or so). the map starts with the first year, in my case mostly 2005 but I want to display the current year (2020)

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin perhaps you can add #1217 also....
